Gateway/Response/PaymentDetailsHandler.php

```diff
@@ -6,6 +6,20 @@
 
 class PaymentDetailsHandler implements HandlerInterface
 {
+    /**
+     * @param string $url URL to Tesco Barcode generated in Omise Backend
```

Comment:
@jacstn (code styling) I saw some code comment block has space between each tag, should we choose one style?
either
```php
/**
 * @param string $url URL to Tesco Barcode generated in Omise Backend
 * @return string Barcode in SVG format
 */
```
or
```php
/**
 * @param  string $url URL to Tesco Barcode generated in Omise Backend
 * @return string Barcode in SVG format
 */
```
or
```php
/**
 * @param  string $url URL to Tesco Barcode generated in Omise Backend
 *
 * @return string Barcode in SVG format
 */
```